### ISSUE TITLE
global descriptions in training dataset

### ISSUE THREAD

**Issue by betterze** [OSU-NLP-Group/MagicBrush#14]
Dear MagicBrush team,

Thank you for sharing this great repo. I really like it. 

In your test set, 'global_descriptions.json' contains global descriptions of both source and target images. However, I do not find the global descriptions in the training set. Could you tell me how to get the global descriptions for the training set?

Thank you for your help.

Best Wishes,
Zongze

**Comment by BennoKrojer** [comment]
Hi! I have asked this before and the recommedation by the authors was to generate it yourself with GPT-4 prompting (see paper). This should be somewhere in an issue here from me.

**Comment by betterze** [comment]
@BennoKrojer Are you referring to Table 8 in the supplementary? To use this method, we need the source prompt, right? There is no source prompt in the training set. 

If I use the image caption model to generate the source prompt, the 'edited concept' in the instruction may be missing or misaligned with the generated caption. For example, given an image of a dining table, the caption network may recognize the sandwich as a hamburger; if the instruction is about the sandwich, then the Table 8 method will fail since there is no sandwich in the source prompt.

Thx a lot for your help.

**Comment by drogozhang** [comment]
Thanks for the question and thanks @BennoKrojer for answering it! I really appreciate your activeness in MagicBrush community :)

Like Benno said, we do not provide the global caption of the training set. Because that's what we used to generate the image with DALLE, we do not plan to release it to avoid potential misuse and to stimulate instruction-guided editing models development, which is the main focus of our benchmark.

For Table 8, the original caption is the caption we can obtain from COCO. The final caption is generated for the edited image. Due to its automated nature, it will include some mismatch issues. I would suggest to use GPT4V or other LMMs like llava to generate that, then you will get high quality caption of the target image.

Best,
Kai

**Comment by betterze** [comment]
get it, thx a lot.
